net-misc/balance/balance-3.57-r1.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs fcaps
+
+DESCRIPTION="TCP Load Balancing Port Forwarder"
+HOMEPAGE="http://www.inlab.de/balance.html"
+SRC_URI="http://www.inlab.de/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="caps"
```

Comment:
`caps` USE is meant for binaries that can conditionally use `sys-libs/libcap`. Here, `filecaps` would be more appropriate. However, I personally think it is not necessary to apply them conditional in this case.